### DISCUSSION TITLE
Month in Revolt - May 2024 - Revolt

### DISCUSSION THREAD

**Discussion by giscus** [stoatchat/discussions#744] [Announcements]
# Month in Revolt - May 2024 - Revolt

Find out what we've been up to in our monthly blog series.

https://revolt.chat/posts/month-in-revolt-may-2024

**Comment by solunareclipse1** [comment]
0/10 too many moths

**Comment by Rexogamer** [comment]
morble

**Comment by bitrain574** [comment]
I don’t have android. But I’ll wait for the iOS app. I can be the best tester to test it out and give feedback.

**Comment by MMachado05** [comment]
Would y'all consider making these available on RSS? Makes reading all of these blog-style updates from various projects I'm following a lot easier! 🙂

**Reply by Rexogamer** [reply]
related: https://github.com/revoltchat/revolt.chat/issues/7

**Comment by arslee07** [comment]
di'dnt read ![rebolt](https://github.com/revoltchat/revolt/assets/50916030/30adefad-d3c8-44a8-8a77-2764ae3a7b5b)

**Reply by Rexogamer** [reply]
I am going to delete your GitHub account

**Reply by ak5o4** [reply]
hilarious
